DVR.xcodeproj/project.pbxproj

```diff
@@ -860,6 +860,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/DVR/Resources/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
let's revert this one to keep fork compatible with the original repo

Reply:
I will do but now it needs to run tests